qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -334,6 +334,82 @@ def test_rmdir_p_dir_with_file(self):
         self.mount_a.stat(self.dir_name)
 
 
+class TestLn(TestCephFSShell):
+    dir1 = 'test_dir1'
+    dir2 = 'test_dir2'
+    dump_id = 11
+
+    def test_soft_link_without_target(self):
+        s = 'somedata'
+        dump_file = 'dump' + str(self.dump_id)
+        test_script_data = f'mkdir -p {self.dir1}/{self.dir2}' \
+                           f'\ncd {self.dir1}\nput - {dump_file}' \
```

Comment:
Let's not use ``put`` (or any other cephfs-shell cmd as far as possible) in tests, so that bugs in ``put`` won't cause unnecessary failure for tests elsewhere?

Reply:
You can use write_file() from mount.py - https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/mount.py#L655